ironfish/src/rpc/routes/wallet/create.test.slow.ts

```diff
@@ -13,7 +13,10 @@ import { RpcRequestError } from '../../clients/errors'
 describe('Route wallet/create', () => {
   jest.setTimeout(15000)
   const routeTest = createRouteTest()
+
   it('should create an account', async () => {
+    jest.spyOn(routeTest.node.wallet, 'scanTransactions').mockReturnValue(Promise.resolve())
```

Comment:
can these all be converted into a single
```
jest.beforeEach(() => {
    jest.spyOn(routeTest.node.wallet, 'scanTransactions').mockReturnValue(Promise.resolve())
})
```

Reply:
updated to use `beforeEach`!